src/Common.props

```diff
@@ -11,7 +11,7 @@
     <OSPlatform>OS_UNKNOWN</OSPlatform>
     <OSArchitecture>ARCH_UNKNOWN</OSArchitecture>
     <DebugConstant></DebugConstant>
-    <VssApiVersion>0.5.234-private</VssApiVersion>
+    <VssApiVersion>0.5.235-private</VssApiVersion>
```

Comment:
We should check these changes don't break downlevel OnPrem versions (e.g. 2022.1 2020, etc).  Ensure checkout works as expected.  Not currently covered by the CI tests.

Reply:
reviewed the server changes; not blocking